waspc/e2e-test/test-outputs/waspBuild-golden/files.manifest

```diff
@@ -52,7 +52,7 @@ waspBuild/.wasp/build/web-app/src/ext-src/Main.css
 waspBuild/.wasp/build/web-app/src/ext-src/MainPage.jsx
 waspBuild/.wasp/build/web-app/src/ext-src/vite-env.d.ts
 waspBuild/.wasp/build/web-app/src/ext-src/vite.config.ts
-waspBuild/.wasp/build/web-app/src/ext-src/waspLogo.png
+waspBuild/.wasp/build/web-app/src/ext-src/waspLogo.svg
```

Comment:
You did search and replace here, but that is not how our e2e tests work. To update them correctly, you actually need to build and run the waspc Haskell project on your machine and update the e2e tests. Which is involved, you need to have Haskell Toolchain running on your machine (GHC, Cabal). We could run these for you possibly though, if you get everything else right! But you should certainly revert these manual changes on e2e tests that you did (everything inside of e2e-test dir).